azure-pipelines.yml

```diff
@@ -1,13 +1,5 @@
 variables:
-    {
-        MODULE_VERSION: "2.1.2",
-        NODE_10: "10.x",
-        NODE_12: "12.x",
-        NODE_14: "14.x",
-        NODE_16: "16.x",
-        NODE_18: "18.x",
-        NODE_20: "20.x",
-    }
+    { MODULE_VERSION: "2.1.2", NODE_14: "14.x", NODE_16: "16.x", NODE_18: "18.x", NODE_20: "20.x" }

```

Comment:
I would leave the tests as-is. Unless the tests are annoying (unreliable or take a super long time), I would only remove them in a new major version (which we already did in the v3.x branch). Keeping them in is an easy way to prevent us from breaking people on old versions and the branch itself will fade out as time goes on

For precedent, the old branch of the worker (v2.x) still tests on Node.js 8/10/12/14: https://github.com/Azure/azure-functions-nodejs-worker/blob/v2.x/azure-pipelines/test.yml#L1

Reply:
+1

Reply:
Done, reverted this change